prow/BUILD.bazel

```diff
@@ -35,19 +40,8 @@ prow_push(
         "sub",
         "tide",
         "tot",
-    ) + image_tags(**{
-        prefix("needs-rebase"): "//prow/external-plugins/needs-rebase:image",
-    }) + image_tags(**{
-        prefix("cherrypicker"): "//prow/external-plugins/cherrypicker:image",
-    }) + image_tags(**{
-        prefix("refresh"): "//prow/external-plugins/refresh:image",
-    }) + image_tags(**{
-        prefix("ghproxy"): "//ghproxy:image",
-    }) + image_tags(**{
-        prefix("label_sync"): "//label_sync:image",
-    }) + image_tags(**{
-        prefix("commenter"): "//robots/commenter:image",
-    }),
+        **_other_images,
```

Comment:
you could also just pass this as a named parameter instead?

Reply:
That screws up *args